### DISCUSSION TITLE
Less Verbose Output In Chat

### DISCUSSION THREAD

**Discussion by kklouzal** [Kilvoctu/aiyabot#198] [Ideas]
Can we have a setting to make the chat output less verbose/minimize chat output of the bot?
It might be nice for some people to see a cute message from the bot when generating an image, followed by a few more lines displaying some current settings.
That being said, can we have an option to disable this verbose output and make it minimalistic? "Generating your image now XXX position in the queue".

**Comment by Kilvoctu** [comment]
Heya, I'll think about it. I like the aiyabot to have personality, so I'm not particularly interested in developing verbosity settings as described here.

**Reply by kklouzal** [reply]
Don't get me wrong, for small servers, the current level of verbosity and the cute little text you get during generation is great. Personally though, I'm usually generating a few hundred images in a single session, if not more, and minimizing the output of the bot down to basically just returning the images I ask for would be very nice to keep channel clutter down.
